packages/components/src/components/checkbox/checkbox.tsx

```diff
@@ -150,7 +150,7 @@ export class Checkbox {
       return (
         <div
           part="helper-text"
-          id={text.id}
+          id={text.internalId}
```

Comment:
I think this is a bug, `text.id` seems correct. (Honestly, I would avoid passing anything to the `renderHelperText` method and get the data needed directly in the method itself.)

Reply:
Oh yes, that's right, I unfortunately slipped in between